decentralised-identity.md

```diff
@@ -0,0 +1,157 @@
+
+### Interoperability:
+
+Interoperability in decentralized identity is like making sure different digital systems speak the same language. It's about ensuring that your digital identity can smoothly work across various platforms and services without any hiccups.
+
+Imagine if your ID card only worked with one type of door lock. That would be inconvenient, right? Interoperability solves this issue. It means your decentralized identity can be recognized and used on different apps, websites, or services, just like your ID works wherever you go.
+
+Imagine buying things online without constantly entering your details. Using blockchain, your payment details become tamper-resistant and transparent. No single authority holds all your financial info, reducing the risk of a massive data breach. This isn't just about security; it's about convenience. With a decentralized identity, online payments become more streamlined, less prone to fraud, and entirely under your control. 
+
+The key is having common standards and rules that everyone follows. This way, whether you're logging into a social media account, accessing a financial service, or using an online marketplace, your decentralized identity seamlessly fits. It's like having a universal key for your digital interactions, making things more convenient and user-friendly. Interoperability is the glue that holds together the vision of a decentralized identity.
+
+### Would security be an issue?
+
+Security remains a critical consideration, primarily as these rely heavily on technologies like blockchain and cryptographic methods. Potential vulnerabilities may arise from implementation flaws, coding errors, or attacks on the underlying infrastructure. The use of cryptographic keys, while fundamental for user control, introduces risks if not adequately managed, with potential concerns related to key loss or theft. Additionally, the distributed and decentralized nature of these systems may pose challenges in ensuring uniform security standards across the entire network.
+
+### Understanding Single Points of Failure and the potential for Eliminating or Not: 
+
+Getting rid of single points of failure in decentralized identity is like making sure there's no one weak link that can mess things up. Think of it as spreading your eggs across different baskets instead of putting them all in one.
+
+In traditional systems, there's often one central authority holding all the keys to your identity. If that authority gets compromised, it's like opening Pandora's box – a single breach can lead to a massive data disaster. Decentralized identity avoids this risk by spreading the responsibility. There's no single database or authority holding all the crucial info. Instead, it's distributed across a network of computers, making it much harder for a single mishap to cause a widespread identity catastrophe.
```

Comment:
I actually think in terms of blockchain data the opposite is true. In essence now it's not a single entity holding your data but many single entities holding your data. Distributed systems basically say, "look here's a database, but to make sure it doesn't get tampered with lets all hold it and coordinate to keep it up to date." 

In many respects, a singe entity holding your data behind a wall is a lot safer than the same information being distributed in public. Also remember, a lot of the same encryption techniques are used in databases as they are in distributed systems. The main difference is, distributed systems are there for the world to see and a single databases is not.

Effectively you've just amped up the chances of being hacked because the hacker knows exactly where to look, because the data is freely available, they can even download the most trusted version.

Reply:
Yes but hopefully we can get to a point that addresses these concerns. I get your point though. Lots of your data spread across traditional systems but then this could also be the case with a decentralised identity. Lots to discuss. I'll leave this piece as it is though as I tried to take the positive view of what it might become.